CHANGELOG.rst

```diff
@@ -1,7 +1,19 @@
+Version 0.2.0 (coming soon...)
```

Comment:
Decided to go with 0.2.0 because it will not be backwards compatible. We didn't increment the minor version last backward compatible patch, but this one will be much more major, it would be better to go to 0.2.0 to signify the changes.